src/utils.jl

```diff
@@ -0,0 +1,21 @@
+# A couple utility methods to avoid messing up var and obs dimensions
+obsdim(dims) = dims
+vardim(dims) = dims == 1 ? 2 : 1
+
+function obswise(data::AbstractMatrix; dims=1)
+    return (selectdim(data, obsdim(dims), i) for i in axes(data, obsdim(dims)))
+end
+
+function varwise(data::AbstractMatrix; dims=2)
+    return (selectdim(data, vardim(dims), i) for i in axes(data, vardim(dims)))
+end
+
+function filterobs(f::Function, data::AbstractMatrix; dims=1)
```

Comment:
Bluestyle suggests:
```suggestion
function filter_obs(f::Function, data::AbstractMatrix; dims=1)
```

Reply:
Again, this is being deprecated and was never exported. I'm inclined to try and minimize how much we're changing deprecated code in this PR. I guess the copying kind of ruined that though :(